lib/galaxy/selenium/navigation.yml

```diff
@@ -213,6 +213,11 @@ multi_history_view:
         selector: '//button[contains(text(), "Copy")]'
 
 
+multiple_histories:
```

Comment:
I know this isn't practiced perfectly - but I feel like the components here should be visual components in the UI - not descriptions of test things? Can you fold these under ``multi_history_view`` higher in the file maybe? The idea is to reuse this file in tours, screenshot generation, tutorial generation, etc.. so the stuff in this file should describe the UI not the tests I think.

Reply:
Good point, it looks cleaner now.